src/ecrc-frontend/src/components/page/transition/Transition.stories.js

```diff
@@ -1,4 +1,6 @@
 import React from "react";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
```

Comment:
is there a reason why we are using router history in stories?

Reply:
because it was complaining about it as the header component has now changed and uses history

Reply:
I think I could also use `MemoryRouter`